crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1951,10 +1939,7 @@ fn test_message_strict_args_count(
         flags | ENABLE_SOFTFORK_CONDITION | ENABLE_MESSAGE_CONDITIONS,
     );
     if flags == 0 {
-        if let Err(e) = ret {
-            println!("{:?}", e);
-        }
-        assert!(ret.is_ok());
```

Comment:
did clippy figure out that this transform is OK? It's pretty impressive in that case

Reply:
Did it by hand here